src/components/Livestream.js

```diff
@@ -55,7 +55,7 @@ const Livestream = () => {
     <Container>
       <TwitchContainer>
         <TwitchFrame
-          src="https://player.twitch.tv/?channel=nwplus&parent=localhost&parent=live.nwplus.io"
+          src="https://player.twitch.tv/?channel=nwplus&parent=localhost&parent=live.nwplus.io&parent=nwplus-ubc.firebaseapp.com"
```

Comment:
Just out of curiosity, Why are you redeclaring the parent query 3 time? Does twitch check each one and take whichever one works, or does it only take the last one?

Reply:
Yeah Twitch will just allow all of them as possible origins

Reply:
cool